extensions/frontend-tiles/src/GraphicsProvider/url-providers/GeoscienceUrlProvider.ts

```diff
@@ -61,11 +61,17 @@ export async function getGeoscienceTilesetUrl(args: GetGeoscienceTilesetArgs): P
   const baseUrl = "https://351mt.api.integration.seequent.com";
```

Comment:
should we be exposing this endpoint?
why not add a temporary arg to GetGeoscienceTilesetArgs to allow you to pass this in?

Reply:
good idea, I did not code that part, but I noticed that the GetGeoscienceTilesetArgs already had an argument to specify a prefix url, so I changed the logic to use it and default to `https://351mt.api.integration.seequent.com` otherwise...

Reply:
i dont want to expose a default endpoint at all

Reply:
done